### ISSUE TITLE
README:  Describe reviewer workflow

### ISSUE THREAD

**Issue by stevegt** [nationofmakers/nationofmakers.github.io#53]
Add a section to the README, walking reviewers through how to set up a personal fork of the repo and review/accept PRs, probably something like:

**Setup**
1. fork repo
2. set github pages config so fork is viewable at https://username.github.io/nationofmakers.github.io/

**Review/Publish PR**
1. pull PR branch to local, push to fork
2. test, edit if needed
3. push any edits back to upstream PR branch
4. if no new edits were needed, merge PR to upstream master -- this will cause github to publish it on the public site
5. if new edits were needed, get someone else to review/merge (can be original submitter if they have merge auth)

**Comment by mkenigson** [comment]
Is this the "dug up, restored, and updated" language described in https://github.com/nationofmakers/nationofmakers.github.io/issues/52 ?

**Comment by stevegt** [comment]
No, this would be a new section in the README.  Also see comment I'm adding right now in #52.
